tests/test_coordinator.py

```
@@ -316,109 +550,115 @@ async def test_retries_forwarded_to_refresh_plant_active(hass, mock_plant):
 
     with patch("custom_components.givenergy_local.coordinator.Client") as mock_cls:
         client = AsyncMock()
         client.connected = True
         client.plant = mock_plant
-        client.refresh_plant = AsyncMock(return_value=mock_plant)
+        client.refresh = AsyncMock(return_value=mock_plant)
+        client.load_config = AsyncMock(return_value=mock_plant)
         mock_cls.return_value = client
         coordinator._client = client
 
         for _ in range(11):  # ticks 0–10
```

Ruff (0.15.14)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)